tsl/src/compression/compression_dml.c

```diff
@@ -203,6 +207,7 @@ decompress_batches_for_insert(const ChunkInsertState *cis, TupleTableSlot *slot)
 		cis->cds->skip_current_tuple = true;
 	}
 
+	cis->cds->batches_deleted += stats.batches_deleted;
```

Comment:
Should we assert this at 0 always for insert decompression?

Reply:
Hmm not sure, MERGE might complicate this